src/app/options/qgscustomprojectionoptions.cpp

```diff
@@ -49,9 +49,6 @@ QgsCustomProjectionOptionsWidget::QgsCustomProjectionOptionsWidget( QWidget *par
   {
     // create an empty definition which corresponds to the initial state of the dialog
     mDefinitions << Definition();
-    QTreeWidgetItem *newItem = new QTreeWidgetItem( leNameList, QStringList() );
```

Comment:
tree widget size/content would then differ from *mDefinitions*, which would probable cause some issues when modifying one or the other (when removing item for instance).

Reply:
Thanks a lot for the feed back. I removed the mDefinitions. It works well for the related issue but maybe this modification will break something else...

Reply:
And what happens if you don't add any new projection but you change the name or the srs widget? Is it working correctly?

Reply:
Old version : the empty row change
Version with my modifications : nothing happens as there is no row in the three. This behavior is, also, not logical. Two ideas : 
- disable the form when there is no row
- or adding a new row in the tree when the name is filled by the user.

For me, it is not so obvious that this is a bug. What do you think ?

Reply:
> disable the form when there is no row

I would go for that solution. It seems the more obvious one